tfhe-csprng/benches/benchmark.rs

```diff
@@ -1,15 +1,17 @@
 use criterion::{black_box, criterion_group, criterion_main, Criterion};
-use tfhe_csprng::generators::{
-    AesniRandomGenerator, BytesPerChild, ChildrenCount, RandomGenerator,
-};
+#[cfg(all(target_arch = "x86_64", target_feature = "aes"))]
+use tfhe_csprng::generators::AesniRandomGenerator as ActivatedRandomGenerator;
+#[cfg(not(all(target_arch = "x86_64", target_feature = "aes")))]
+use tfhe_csprng::generators::SoftwareRandomGenerator as ActivatedRandomGenerator;
```

Comment:
I don't think this is what we want, if we want to bench something we should have control on the implementation, historically there was only the x86 generator hence the old import

Reply:
How do you want to control it ? 
In that case I think it makes sense to follow @tmontaigu suggestion and move the ActivatedRandomGenerator definition in csprng, so that they are controlled the same way. csprng still exports all possible generators, it's only the type alias that changes

Reply:
In that case in the CSPRNG could be "FastetRandomGenerator" for the alias

Reply:
"DefaultRandomGenerator", maybe ?

Reply:
sure